stencil-library/src/components/payment-form/payment-form.tsx

```diff
@@ -103,6 +103,7 @@ export class PaymentForm {
               }}
             />
           </div>
+          <slot name='insurance' />
```

Comment:
Do you see any css issues related to inherited styles since this is a direct child of an element with the `row gy-3` classes? I am wondering if it should be wrapped in a grid element, however if there is nothing in the slot we don't want a random space. Not sure what the best way to handle this is off the top of my head

Reply:
Interesting, I hadn't considered that to be honest.

Now that I look at how it works, it seems the slot doesn't inherit anything except from :host
<img width="741" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/10523683/40029942-0b25-4a3d-865e-bcdba6af62b0">

Reply:
This can be a problem, currently it inherits the following:
<img width="1483" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/10523683/3599ca6f-b3c5-46bb-b744-dc9d3611073d">
I don't think that will mess anything up, but we can definitely try it out and iterate if it's an issue.